src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,95 @@
+package duke.command;
+
+import duke.task.Todo;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.ui.Storage;
+import duke.ui.TaskList;
+import duke.ui.Ui;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Adds a new task.
+ */
+public class AddCommand extends Command {
+    private String[] oneLine;
+
+    public AddCommand(String[] oneLine) {
+        assert oneLine != null;
+        this.oneLine = oneLine;
+    }
+
+    /**
+     * check if it is a valid time format.
+     *
+     * @param str The input should be checked.
+     * @return true if str can be converted to a valid time format, otherwise returns false.
+     */
+    public static boolean isValidTime(String str) {
+        assert str != null;
+
+        try {
+            DateTimeFormatter formatter1 = DateTimeFormatter.ofPattern("dd/MM/yyyy HHmm");
```

Comment:
The method argument `str` is not being used in the method, perhaps you may have missed a line of code?